src/createDocumentActionDuplicate.js

```diff
@@ -0,0 +1,57 @@
+import sanityClient from 'part:@sanity/base/client'
+import { CopyIcon } from '@sanity/icons'
+import { i18n as getI18n } from './i18n'
+import { useRouter } from '@sanity/base/router'
+import pluginConfig from 'config:@kaliber/sanity-plugin-duplicate'
+
+const client = sanityClient.withConfig({
+  apiVersion: '2022-04-06'
+})
+
+export function createDocumentActionDuplicate(documentSchemes) {
+  const documentTypes = getReplacementFunctionsForAllSchemes(documentSchemes)
+  const i18n = getI18n(pluginConfig.language)
+
+  return function DocumentActionDuplicate({ type, published, draft }) {
+    const router = useRouter()
+    return {
+      icon: CopyIcon,
+      label: 'Duplicate',
+      title:  i18n['duplicate'],
```

Comment:
Je hoeft hier geen rekening te houden met Taal. We kunnen de andere buttons ook niet aanpassen + we hebben afgesproken dat we alles in het beheer in het Engels doen totdat Sanity zelf i18n ondersteund.

Maakt het ook meteen makkelijker omdat dan de config weg kan en de library niet meer als plugin geïnstalleerd hoeft te worden.

Reply:
Kan trouwens ook geen kwaad om het te hebben. Zie wel dat die andere plugins het nog wel hebben...